### DISCUSSION TITLE
Where to find all CommonStyles?

### DISCUSSION THREAD

**Discussion by EmrecanKaracayir** [microsoft/WindowsAppSDK#2081] [General] [answered]
Is there a list of all common styles? For example: 
"SystemFillColorSuccessBackgroundBrush"
"SystemFillColorSuccessBrush"

And also their values?

For example I want to learn which brushes are used in InfoBar, where can I find them?

**Comment by kmgallahan** [comment, marked as answer]
Controls that are open source have styles in their folder [here](https://github.com/microsoft/microsoft-ui-xaml/tree/main/dev).

Closed source are [here](https://github.com/microsoft/microsoft-ui-xaml/tree/main/dev/CommonStyles).

`*_V*.xaml` files are older versions. Some style elements are in the `*_themeresources.xaml` files and the others in `*.xaml` files (example: `InfoBar.xaml` & `InfoBar_themeresources.xaml`).

Then you have [`Common_themeresources_any.xaml`](Common_themeresources_any.xaml) & [`Common_themeresources.xaml`](Common_themeresources.xaml) which are grab bags of common styles.
